rendercv/data_models.py

```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
         "projects": [
             NormalEntry(
                 name="Multi-User Drawing Tool",
-                date="2004",
+                date="2004",  # I would omit start/end dates and replace with a project link as the date is largely irrelevant
```

Comment:
Maybe to showcase that RenderCV's date field doesn't need to be dates, we can do this:

date="[github.com/user/example](https://github.com/user/example)"

Also, we shouldn't leave any discussions as comments in the source code.

Reply:
Yes, you can replace with that.